README.adoc

```diff
@@ -45,3 +45,8 @@ Deployment Guide::
 
 .*No commits to master*
 NOTE: Please do not make any commits to the `master` branch. `master` is reserved for releases only. 
+
+Building documentation
+----------------------
+
+If you're contributing to the cloud documentation in this repo and want to build using our DAPs tooling, see the DAPs Quickstart for more information: http://docserv.nue.suse.com/documents/DAPS/daps-quick/html/
```

Comment:
We should make reference to: https://github.com/openSUSE/daps2docker
This will build daps docs (defaults to html and pdf) and all you need is to install docker. Then run: 
`./daps2docker.sh /PATH/TO/DOC-DIR`
OR
`./daps2docker.sh /PATH/TO/DC-FILE`

If someone wants to simply build some DAPs docs rather then more fully develop them then this makes the bar much lower :)
Just a thought :)

Reply:
Good suggestion :) thanks!